src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/HtmlHelperOfT.cs

```diff
@@ -29,12 +29,12 @@ public class HtmlHelper<TModel> : HtmlHelper, IHtmlHelper<TModel>
             UrlEncoder urlEncoder,
             ExpressionTextCache expressionTextCache)
             : base(
-                htmlGenerator,
-                viewEngine,
-                metadataProvider,
-                bufferScope,
-                htmlEncoder,
-                urlEncoder)
+                  htmlGenerator,
+                  viewEngine,
+                  metadataProvider,
+                  bufferScope,
+                  htmlEncoder,
+                  urlEncoder)
```

Comment:
Not sure why this changed. It looks more like what VS does but meh. I'll revert if you prefer.